docs/include/conf/portland/team.rst

```diff
@@ -0,0 +1,67 @@
+Folks
```

Comment:
I don't think it makes sense to include this. 

The bios are already included, and this way whenever we change it next year it'll update other years also.

Reply:
I guess I was trying to create a situation where each conference has a different include and it magically changes for Portland, Prague, etc. But now I see that I'm probably missing an IF statement somewhere :) if it's too much of a PITA I can just revert it back to hardcoded content...

Reply:
yeah, even that way though, it'll break. I reckon leave this one hard coded.

As an alternative, :smiling_imp: `conf.yaml`

```yaml
roles:
  chair: eric
  speakers: sam
```

but I don't _think_ it's worth it for this.